src/AsyncBridge/Properties/SpecificAssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Security;
+
+#if !NET35
+[assembly: AllowPartiallyTrustedCallers]
+#if !PORTABLE
+[assembly: SecurityRules(SecurityRuleSet.Level2)]
```

Comment:
When this line is removed, no tests fail. Is it possible to test this in some way? If not, can you add a comment above it explaining why it is needed?

Also, no biggie but would you mind renaming the file to AssemblyInfo.cs?

Reply:
The SecurityRules is recommended to be added. On my phone so I can't reference the exact reason. Will check later.

Will rename. That's what the file was called in the older project before the cleanup.

Reply:
https://docs.microsoft.com/en-us/dotnet/framework/misc/security-transparent-code-level-2#usage-examples-and-behaviors

"If you do not annotate an assembly, the .NET Framework 4 rules are used by default. However, the recommended best practice is to use the SecurityRulesAttribute attribute instead of depending on the default."

Reply:
It looks like it's locking it rather than following the CLR default. That's fine so long as we can be sure that this doesn't cause any issues on CLR v2, which we don't run tests against.

Reply:
It looks like it's locking it rather than following the CLR default. That's fine so long as we can be sure that this doesn't cause any issues on CLR v2, which we don't run tests against. How would we verify that?

Reply:
Does the project even build against a vanilla 2.0? The attribute only exists in 4.0+, so maybe we just need to change the #if to specifically check for 4.0+